Snail/Extensions/UIViewControllerExtensions.swift

```diff
@@ -0,0 +1,20 @@
+//  Copyright © 2016 Compass. All rights reserved.
+
+#if os(iOS) || os(tvOS)
+import UIKit
+#else
+import Foundation
+#endif
+
+extension UIViewController {
+    private static var disposerKey = "com.compass.Snail.UIViewController.disposer"
+
+    public var disposer: Disposer {
```

Comment:
since this is a `UIViewController` extension could this just be...
```swift
public var disposer: Disposer {
    return view.disposer
}
```?

Reply:
We can ... try. It doesn't feel wrong, but feels to me one of those things that you change before merge and you never got to fully test

Reply:
Ok, we can do that and it works. It would create a dependency between both extensions. I'm going to go ahead and make the change.